Tools/autotest/arducopter.py

```diff
@@ -8841,6 +8877,9 @@ def tests1c(self):
              "Test Splines and Terrain",
              self.test_terrain_spline_mission),
 
+            ("MotorFailsafe",
```

Comment:
.. and here too.  For me at least, it seems more similar to an arming check than a failsafe.

Reply:
Fixed